MhLabs.Calendar/Calendar.cs

```diff
@@ -89,5 +89,28 @@ private static string ToClientFormat(DateTime dateTime, string format, string ti
             return result;
         }
 
+        /// <summary>
+        /// <para>Convert dateTime string to UTC using specified time zone</para>
+        /// <para>The following example will convert Swedish datetime: 2020-06-30 15:35 to Universal datetime: 2020-06-30 13:35</para>
+        /// <para></para>
+        /// <code>
+        ///   Calendar.ConvertToUniversalTime("2020-06-30 15:35", TimeZones.Sweden);
+        /// </code>
+        /// </summary>
+        /// <param name="dateTime"></param>
+        /// <param name="timeZone"></param>
+        /// <example>Hello world: <code>var x = y;</code></example>
+        public static DateTime ConvertToUniversalTime(string dateTime, string timeZone)
+        {
+            var parsedDateTime = ParseAsLiteral(dateTime);
+            var clientFormat = ToRoundTripDateTime(parsedDateTime, timeZone);
+
+            if (!DateTime.TryParse(clientFormat, out DateTime result))
```

Comment:
DateTimeOffset.TryParse, DateTime does not handle offset included in roundtrip format in a reliable manner